### ISSUE TITLE
Fix triagebot cron job

### ISSUE THREAD

**Issue by mgreensmith** [mozilla-itcloud/maintenance-project#10]
Traigebot has a cron job that queries a Bugzilla API endpoint that doesn't exist after Bugzilla migrated to AWS.

Need to fix the cron to point to the new Bugzilla URI.

**Comment by mgreensmith** [comment]
```
[root@appsvcs-generic/us-west-2/core/stage {triagebot.nubis} triagebot]$ python dashboard.py
2018-07-11 21:00:08 UTC - INFO - Generating static audit page.
Traceback (most recent call last):
 File “dashboard.py”, line 134, in <module>
   main()
 File “dashboard.py”, line 53, in main
   bugdays[d] = getOutstandingBugs(bzagent, d)
 File “dashboard.py”, line 126, in getOutstandingBugs
   bugs = bzagent.get_bug_list(options)
 File “/usr/local/lib/python2.7/dist-packages/bugzilla/agents.py”, line 30, in get_bug_list
   return BugSearch.get(url).bugs
 File “/usr/local/lib/python2.7/dist-packages/remoteobjects/fields.py”, line 141, in __get__
   value = obj.api_data[self.api_name]
 File “/usr/local/lib/python2.7/dist-packages/remoteobjects/promise.py”, line 136, in _get_api_data
   self.deliver()
 File “/usr/local/lib/python2.7/dist-packages/remoteobjects/promise.py”, line 216, in deliver
   self.update_from_response(request[‘uri’], response, content)
 File “/usr/local/lib/python2.7/dist-packages/remoteobjects/promise.py”, line 232, in update_from_response
   super(PromiseObject, self).update_from_response(url, response, content)
 File “/usr/local/lib/python2.7/dist-packages/remoteobjects/http.py”, line 263, in update_from_response
   self.raise_for_response(url, response, content)
 File “/usr/local/lib/python2.7/dist-packages/remoteobjects/http.py”, line 243, in raise_for_response
   % (classname, url, response.get(‘content-type’)))
remoteobjects.http.BadResponse: Bad response fetching BugSearch https://api-dev.bugzilla.mozilla.org/latest/bug/?changed_before=2018-07-10&status=UNCONFIRMED&product=Firefox&changed_field=%5BBug%20creation%5D&changed_after=2018-07-10&component=Untriaged&username=triagebot-api%40mozilla.bugs&password=PASSWORDHERE: content-type text/html; charset=UTF-8 is not an expected type
```

**Comment by mgreensmith** [comment]
https://files.slack.com/files-pri/T027LFU12-FBP5BK38V/screen_shot_2018-07-11_at_2.02.11_pm.png

**Comment by mgreensmith** [comment]
TL;DR: content-type mismatch fetching `Bad response fetching BugSearch https://api-dev.bugzilla.mozilla.org/latest/bug/?changed_before=2018-07-10&status=UNCONFIRMED&product=Firefox&changed_field=%5BBug%20creation%5D&changed_after=2018-07-10&component=Untriaged&username=triagebot-api%40mozilla.bugs&password=PASSWORDHERE`, mhoye knows abiout it and is working on it
